test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -494,4 +505,57 @@ public void onCommit(Map<String, List<CorfuStreamEntry>> mutations) {
         }
         log.debug(table.getMetrics().toString());
     }
+
+    /**
+     * Validate that nested transactions do not throw exception if txnWithNesting is used.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void checkNestedTransaction() throws Exception {
+// Get a Corfu Runtime instance.
+        CorfuRuntime corfuRuntime = getTestRuntime();
```

Comment:
**Nit:** Can we indent the comment with the parameter?

Reply:
done. thanks